### DISCUSSION TITLE
I need to change my .wslconfig files and need help with modification to change the number of CPUs and RAM used by Docker

### DISCUSSION THREAD

**Discussion by helpWithCoding** [microsoft/WSL#9574] [General]
hi, I am trying to this part (was recommended to come here to github for help): https://degauss.org/troubleshooting.html#Parallel_Computing_on_Windows and need to change the number of CPUs used to get the program to work. However, I am new to this and have followed this guide (https://learn.microsoft.com/en-us/windows/wsl/wsl-config#configure-global-options-with-wslconfig) but it is very technical for me and do not want to compromise my system when doing so. I cannot change the resources in Docker so I need to change the .wslconfig file to do so. I am running Ubuntu subsytem on windows 11

**Comment by MacyFonseca** [comment]
My take in this problem was follow this steps:

- Locate wslconfig file in `C:\Users\<you_username>\wslconfig` (if not found you can create it)
- Place the following in the file:
      [wsl2]
      memory=4GB # How much memory to assign to the WSL2 VM.
      processors=2 # How many processors to assign to the WSL2 VM.
- Right click in docker icon and press Restart

You can always check this [link](https://medium.com/geekculture/how-to-limit-memory-usage-on-docker-desktop-wsl-2-mode-2a4a719f05fd) fro additional info.

**Reply by FLAMESpl** [reply]
what's the docker icon?

**Reply by olivsinz** [reply]
More details: https://learn.microsoft.com/en-us/windows/wsl/wsl-config#configuration-settings-for-wslconfig

**Comment by mhekel** [comment]
it seems people assume that everyone runs docker for everything. ... no thanks. wsl2 does just fine with out it
